MaraBot/Messages/Display.cs

```diff
@@ -34,10 +34,10 @@ public static class Display
         /// <param name="ctx">Command Context.</param>
         /// <param name="preset">Preset used in race.</param>
         /// <param name="seed">Generated seed.</param>
-        /// <returns>Returns an asynchronous task.</returns>
-        public static Task RaceAsync(CommandContext ctx, Preset preset, string seed)
+        /// <returns>Returns an embed builder.</returns>
+        public static DiscordEmbedBuilder RaceEmbed(CommandContext ctx, Preset preset, string seed)
         {
-            return RaceAsync(ctx, preset, seed, DateTime.Now);
+            return RaceEmbed(ctx, preset, seed, DateTime.Now);
         }
```

Comment:
Can't we add this overload to the main function by giving `timestamp` a default value of `DateTime.Now`?

Reply:
No, you'll get this error if you do:
`Default parameter value for 'timestamp' must be a compile-time constant`

Reply:
Ah I see. I had a slight feeling that could be the case, not thát proficient in C# lol... the more you know! :D